src/HomeWork4/Classes/Author.java

```diff
@@ -0,0 +1,37 @@
+package HomeWork4.Classes;
+
+public class Author extends Book{
+    private String name;
+    private String lastName;
+    private int age;
+    private String country;
+    public void setAuthor(String name, String lastName, int age,String country){
+        this.name = name;
```

Comment:
for what we use this method?

Reply:
I want to set all information about Author and work with this information.